app/components/LeftPaneEntries/index.tsx

```diff
@@ -352,6 +467,65 @@ function LeftPane(props: Props) {
         onEntryRestore,
         entryImagesMap,
         isEntrySelectionActive,
+        entryAttachmentsMap,
+        leadAttachmentsMap,
+    ]);
+
+    const leadAttachmentItemRendererParams = useCallback((
+        _: string,
+        attachment: LeadPreviewAttachmentType,
+    ): TablesAndVisualsItemProps => {
+        const entry = entriesMappingByAttachment[attachment.id];
+        if (entry) {
+            const entryId = entry.clientId;
+            return {
+                type: 'entry-item' as const,
+                ...entry,
```

Comment:
Is there a reason we have the type before the object spread?

Reply:
Oops. No reason!